recipes/snorkel/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "snorkel" %}
+{% set version = "v0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/HazyResearch/{{ name }}/archive/{{ version }}-beta.tar.gz
+  sha256: 483eca03494255474452a97a0e13b27868f0fadaecb4bfc2c7610e28f54d87d8
+
+build:
+
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3.6
+    - pip
+  run:  #https://github.com/HazyResearch/snorkel/blob/master/environment.yml
+    - python >=3.6
+    - ipywidgets =7.4.2
+    - pytorch ==1.1.0
+    - beautifulsoup4 ==4.7.1
+    - future ==0.17.1
+    - jupyter ==1.0.0
+    - lxml ==4.3.3
+    - matplotlib ==3.0.1
```

Comment:
There is a great talk from msarahan where he discusses that packages shouldn't pin version requirements.

I think it is this one: https://www.youtube.com/watch?v=Kamld5Z-xx0

TLDL (too long don't listen) If you pin your version requirements so harshly, you aren't creating a package, but more of a distrubtion, where people aren't allowed to mix and match versions and where you will be unable to upgrade.

There is virtually no reason to pin so hard. many packages. Most o the packages I see here are actually quite popular packages that take care not to change so drastically between patch versions. Conda forge will often not even support slightly older versions when pinnings move forward.

Reply:
I agree on this. I'm helping on bringing a package with a strict [requirement](https://github.com/HazyResearch/snorkel/blob/master/environment.yml#L5-L26).